src/routes/router.ts

```diff
@@ -68,10 +68,14 @@ RootRouter.get("/create-branch-options", GithubCreateBranchOptionsGet);
 RootRouter.use("/github", GithubRouter);
 RootRouter.use("/jira", JiraRouter);
 
-// On base path, redirect to Github App Marketplace URL
-RootRouter.get("/", async (req: Request, res: Response) => {
-	const { jiraHost, gitHubAppId } = res.locals;
-	const gitHubAppClient = await createAppClient(req.log, jiraHost, gitHubAppId);
+// On base path, redirect to Github App Marketplace URL (This is only for cloud)
+type RedirectResponseType =  Response<
+	never,
+	JiraHostVerifiedLocals
+>;
+RootRouter.get("/", async (req: Request, res: RedirectResponseType) => {
```

Comment:
Let me know if I am wrong, this route ONLY works for cloud right? We don't have intention for it to work for GHE as we won't have uuid in the path?

Reply:
This is the top level route of the domain (`/`). as such, it has no knowledge of GHE instances is it's not under the `/github` route